### ISSUE TITLE
deploy error uwp program

### ISSUE THREAD

**Issue by bigdimboom** [Microsoft/cppwinrt#191]
I got xbox ystem update recently. My program winrtcpp program stops working, while other c++/cx program can still run on the xbox. In the meantime, I did not change my code. The code is still running fine on PC 

![image](https://cloud.githubusercontent.com/assets/6119906/25826402/c0d89db0-3414-11e7-86eb-92cf3ce26448.png)

**Comment by BrentRe** [comment]
You should bring this up through your Xbox support channels as this is not a problem specific to C++/WinRT.

**Comment by walbourn** [comment]
Is this a UWP on Xbox using the retail development mode?

**Comment by bigdimboom** [comment]
no, it is an app that I wrote and I'm debugging in XBOX dev mode. Like I described, it was still working on last Friday before I went home. Today I have problem of debugging. The only difference I can see is that I run the update pushed by the system in dev mode. And I have tried everything I have that is used to be able to solve the problem but they are working this time. e.g. "clear cache" , "reboot", "repair visual studio", "change package names".
